apps/web/lib/dictionary.tsx

```diff
@@ -710,6 +710,11 @@ const dictionary = {
         fr: "Solde total",
       },
     },
+    buttonGroupLabels: {
+      all: { en: "View all", pl: "Wszystkie", fr: "Voir tout" },
+      income: { en: "Income", pl: "Wpływy", fr: "Les recettes" },
```

Comment:
I know design says "Income" but since `pl` and `fr` are in plural form here, and `Expenses` below are also in plural, I would also change it here.
```suggestion
      income: { en: "Incomes", pl: "Wpływy", fr: "Les recettes" },
```

Reply:
I agree :)

Reply:
done